components/atom/progressBar/src/ProgressBarLine/index.js

```diff
@@ -14,6 +14,7 @@ import useIndicator from './useIndicator.js'
 import usePercentage from './usePercentage.js'
 
 const ProgressBarLine = ({
+  hideBorder,
```

Comment:
some _positive_ boolean prop (say `borderless`) would probably make for better naming IMHO... what do you think?

Reply:
I was trying to follow the same pattern, as the property `hideIndicator` already exists. Thoughts?

Reply:
yeah saw that... the difference I see is perhaps `hideIndicator` makes more sense as an _action_ word rather than a  _boolean-ish_ named prop as the _Indicator_ seems to be a more substantial piece of the component, whereas displaying it _borderless_ or not is something that's just related to styles... if that makes sense? both are fine anyway...